docs/ReleaseNotes.md

```diff
@@ -16,6 +16,9 @@ The FoundationDB Java binding dependency has been updated to 7.1 with this relea
 
 This release also updates downstream dependency versions. Most notably, the proto3 artifacts now require Protobuf version 3.20.1.
 
+`OnlineIndexer` no longer supports changing the initial delay & max delay used when retrying, in the middle of an ongoing
```

Comment:
I'm not sure what the correct way to do this is, but I feel like it should be mentioned, perhaps it warrants a new minor build?